lib/debuglet.js

```diff
@@ -30,9 +31,9 @@ var StatusMessage = require('./apiclasses.js').StatusMessage;
 
 var assert = require('assert');
 
-var NODE_VERSION_MESSAGE = 'V8 Debug API not available – make sure you are ' +
-    'running Node.js >=0.12. Cloud Debugger agent will not be able to take ' +
-    'snapshots.';
+var NODE_VERSION_MESSAGE = 'An unsupported version of Node.js is running. ' +
+  'Cloud Debugger is only supported with Node.js >=0.12 with the exception of ' +
+  ' 5.2.';
```

Comment:
How about rewording this to 'Node.js version not supported. Node.js 5.2.0 and versions older than 0.12 are not supported.'